tests/integration/tier0/non-destructive/firewalld-inhibitor/test_firewalld_inhibitor.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from envparse import env
+
+from convert2rhel.actions.system_checks.check_firewalld_availability import FIREWALLD_CONFIG_FILE
+
+
+@pytest.mark.test_firewalld_inhibitor
+def test_firewalld_inhibitor(shell, convert2rhel):
+    """
+    Verify that on the OL8.8 the conversion is inhibited due to
+    running firewalld on the system and the `CleanupModulesOnExit` configuration
+    option is set to `yes` in firewalld configuration file.
+    The reference ticket: https://issues.redhat.com/browse/RHELC-1180
+    """
+    assert shell("rpm -q firewalld").returncode == 0
+    assert shell("systemctl start firewalld").returncode == 0
+    assert shell("systemctl enable firewalld").returncode == 0
+
+    assert shell(f"grep 'CleanupModulesOnExit=yes' {FIREWALLD_CONFIG_FILE}").returncode == 0
+
```

Comment:
This might be tricky, I mean we have more or less the system under our control, but if for whatever reason the Oracle changes this, we won't match and the test will fail too soon.
I would either backup the original and provide our own config with the option disabled, or try regex matching more variants.